src/flask_sqlalchemy/__init__.py

```diff
@@ -172,7 +172,14 @@ def __init__(self, db, autocommit=False, autoflush=True, **options):
             **options,
         )
 
-    def get_bind(self, mapper=None, **kwargs):
+    def get_bind(
+        self,
+        mapper=None,
+        clause=None,
+        bind=None,
+        _sa_skip_events=None,
+        _sa_skip_for_implicit_returning=False,
```

Comment:
Why is `**kwargs` not sufficient here? Also, if we're going to capture these, why not pass them on in `super().get_bind()`?

Reply:
That's a very good question, why `**kwargs` is not sufficient.

Basically I found that:

``` python
from flask import Flask
from flask_sqlalchemy import SQLAlchemy, SignallingSession
from sqlalchemy.orm.session import Session
from sqlalchemy.orm import sessionmaker


class ExpSession(SignallingSession):
    def get_bind(self, *args, **kwargs):
        print(args, kwargs)


class ExpSQLAlchemy(SQLAlchemy):
    def create_session(self, options):
        return sessionmaker(class_=ExpSession, db=self, **options)


app = Flask(__name__)
app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///example.sqlite"
db = ExpSQLAlchemy(app)

session = db.session

print(session.get_bind())
```

The output is:
```
(None, None, None, None, False) {}
```

Reply:
@dpgaspar The example you show above seems to behave the same with and without your PR.